scripts/dump_prims.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 SiFive, Inc.
```

Comment:
```suggestion
#!/usr/bin/env python3

# Copyright 2023 SiFive, Inc.
```

Reply:
I wondering if we actually need this script to be retained, as it's rare to add a new prim, and it could be manually added to prim.wake at that time

Reply:
Yeah, fair. I certainly had that thought too